index.php

```diff
@@ -237,7 +242,7 @@ function get_mediatype_sql($m)
     This is seperately assigned as a LIMIT so, if this exists, 
     lets just skip page numbers and carry on
 */
-if (!$LIMIT && ($config['pageno'] > 0) &! ($pageno == 'all'))
+if ($LIMIT == '' && ($config['pageno'] > 0) &! ($pageno == 'all'))
```

Comment:
Even though this might work the same (I think) I'd suggest keeping the original

Reply:
as `$LIMIT` may not have been set by this code is executed i add an init at line 124.
this stops 'Undefined variable $LIMIT' at lines 245 & 297